packages/babel/src/index.mjs

```diff
@@ -91,16 +91,16 @@ export default function (babel, opts = {}) {
         }
         const calleePath = path.get('callee');
         switch (calleePath.node.type) {
+          case 'CallExpression':
           case 'MemberExpression':
           case 'Identifier': {
-            const calleeSource = calleePath.getSource();
-            const firstArgPath = argsPath[0];
+            const calleeSource = calleePath.getSource().split('(')[0];
```

Comment:
In a `CallExpression`, the `getSource()` function returns the callers name... But also any parameters used to create the inlined anonymous function, such as `withDisplayName("SomeDisplayName")`.

By splitting at the first found `(` and taking whatever comes before, I get only the last named function before any currying.

I'm not sure if that is needed, but it felt cleaner than dumping all the arguments in the generated key.